index.html

```diff
@@ -0,0 +1,58 @@
+<!DOCTYPE html>
+
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <title>Portfolio</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="style.css">
+    </head>
+
+    <body>
+        <header>
+            <nav class="nav_container">
+                <div class="logo">
+                     <img src="images/MyLogo.png" alt="Logo">
+                </div>
+     
+                <div class="toggle">
+                    <img src="images/Union.png" alt="Logo">
+                </div>
+             </nav>
```

Comment:
- [ ] Please consider removing the `nav` tag which you used to wrap the logo and the toggle as we need the `nav` tag to only wrap the toggle. You might replace it with a `div` tag.